ViewControllers/MovieDetail/StackMovieDetailView.swift

```diff
@@ -0,0 +1,171 @@
+        label.font = UIFont.systemFont(ofSize: 15)
+        label.textColor = UIColor(named: "staticTextColor")
+        label.text = "Released on"
+        label.textAlignment = .left
+        return label
+    }()
+    
+    private(set) lazy var releaseDateLabel: UILabel = {
+        let label = UILabel()
+        label.font = UIFont.systemFont(ofSize: 15)
+        label.textColor = UIColor(named: "descriptionColor")
+        label.textAlignment = .left
+        return label
+    }()
+    
+    private(set) lazy var lastsLabel: UILabel = {
+        let label = UILabel()
+        label.setContentHuggingPriority(.defaultHigh, for: .horizontal)
+        label.font = UIFont.systemFont(ofSize: 15)
+        label.textColor = UIColor(named: "staticTextColor")
+        label.text = "Lasts"
+        label.textAlignment = .left
+        return label
+    }()
+    
+    private(set) lazy var durationLabel: UILabel = {
+        let label = UILabel()
+        label.font = UIFont.systemFont(ofSize: 15)
+        label.textColor = UIColor(named: "descriptionColor")
+        label.textAlignment = .left
+        return label
+    }()
+    
+    private(set) lazy var movieDescriptionLabel: UILabel = {
+        let description = UILabel()
+        description.numberOfLines = 0
+        description.font = UIFont.systemFont(ofSize: 15)
+        description.textColor = UIColor(named: "descriptionColor")
+        description.textAlignment = .left
+        return description
+    }()
+    
+    private(set) lazy var collectionViewFlowLayout: UICollectionViewFlowLayout = {
+        let layout = LeftAlignedCollectionViewLayout()
+        layout.minimumInteritemSpacing = 4
+        layout.minimumLineSpacing = 4
+        layout.sectionInset = UIEdgeInsets(top: 4, left: 0, bottom: 0, right: 0)
+        layout.estimatedItemSize = UICollectionViewFlowLayout.automaticSize
+        return layout
+    }()
+    
+    private(set) lazy var collectionView: IntrinsicallySizedCollectionView = {
+        let collectionView = IntrinsicallySizedCollectionView(frame: .zero, collectionViewLayout: collectionViewFlowLayout)
+        collectionView.register(DetailCollectionViewCell.self, forCellWithReuseIdentifier: DetailCollectionViewCell.reuseIdentifier)
+        collectionView.isScrollEnabled = false
+        collectionView.backgroundColor = UIColor(named: "backgroundColor")
+        return collectionView
+    }()
+    
+    private(set) lazy var scrollView: UIScrollView = {
+        let scrollView = UIScrollView()
+        scrollView.translatesAutoresizingMaskIntoConstraints = false
+        return scrollView
+    }()
+    
+    private(set) lazy var scrollContentStack: UIStackView = {
+        let stack = UIStackView()
+        stack.translatesAutoresizingMaskIntoConstraints = false
+        stack.axis = .vertical
+        stack.layoutMargins = UIEdgeInsets(top: 0, left: 16, bottom: 0, right: 16)
+        stack.isLayoutMarginsRelativeArrangement = true
```

Comment:
If you hadn't set this property to 'true', would all views be left-aligned? or what would happen?

Reply:
I need to set `isLayoutMarginsRelativeArrangement` to the collection to propagate the `layoutMargins` that I set above it. By default the stack will layout its arranged subviews according to its own bounds (entire view), if I set it to true then it layout its arranged subviews following the layout margins